deploy/airgap-prepare.sh

```diff
@@ -71,10 +79,11 @@ rm $K3S_INSTALL_SCRIPT \
 	$K3S_BINARY \
 	$K3S_IMAGES_TAR \
 	$CRED_SHIELD_IMAGES_TAR \
+	$CERT_MANAGER_IMAGES_TAR \
 	${DIST}/$CERT_MANAGER_MANIFEST \
 	${DIST}/$CERT_MANAGER_CONFIG_MANIFEST \
 	${DIST}/$CERT_MANIFEST \
-  ${DIST}/$CRED_SHIELD_DEPLOYMENT_MANIFEST \
+    ${DIST}/$CRED_SHIELD_DEPLOYMENT_MANIFEST \
```

Comment:
Minor but can we fix the indentation to be the same

Reply:
I did add extra spaces, but it seems it still didn't fix properly.